.gitignore

```diff
@@ -5,9 +5,10 @@ node_modules
 dist
 .parcel-cache
 coverage
-.idea
+../.idea
```

Comment:
Looks like this can go, right? No files in the parent directory should be tracked anyway 🤔